roles/acm_setup/README.md

```diff
@@ -42,6 +46,29 @@ See below an example of how to use the acm_setup role to configure ACM.
       hub_disable_selfmanagement: true
       hub_availabilityConfig: High
       hub_disconnected: true
+      hub_os_images:
+        - openshiftVersion: "4.15.0-0.nightly-2024-04-21-051624"
+          version: "4.15"
+          url: "http://webcache.dfwt5g.lab:8080/rhcos-415.92.202402201450-0-live.x86_64.iso"
+          cpuArchitecture: x86_64
+```
+
+## OS images
+
+```yaml
+hub_os_images:
```

Comment:
Are these items indented properly, compared to the previous one?

Reply:
yes, as those entries are examples only of how to define the OS images. I added some comments about it. Thanks!